### PR TITLE
EZP-24916: Platform UI Preview broken

### PR DESCRIPTION
**Pull request by dpobel**
JIRA: https://jira.ez.no/browse/EZP-24916

# Description

This patch fixes the display of the preview. With it, we also handle better handle the preview on a device bigger than the one used by the editor.

Screencast: http://youtu.be/NeOrEjiODWA

# Tests

unit tests + manual tests

### CODE REVIEW

**Comment by andrerom** [comment]
you'll also need to set the hight to size of screen right? otherwise it will inherit?

**Comment by dpobel** [comment]
well, you are right we have size issues, if we don't set the height like in the current patch, the ez-editpreviewview-container element will take the height needed to display its content but this can be more than the viewport height (and then the bottom of the preview can not be reached) or this can be less than the viewport (and in this case that's ugly). The thing is I can force this element to be the exact viewport height but this leads to the question I already ask several times, how do we handle the case where the preview is supposed to be wider or higher than the actual viewport size ?

**Comment by dpobel** [comment]
ping @StephaneDiot @yannickroger @mhyndle @andrerom

**Comment by mhyndle** [comment]
+1

File: Resources/public/css/theme/views/editpreview.css [hunk 1]
```diff
@@ -5,34 +5,61 @@
...
     background: #f3f3f3;
+    color: #555;
 }
 
-.ez-preview-content .ez-preview-hide {
-    color: #555;
+.ez-view-editpreviewview .ez-preview-iframe-wrapper:before {
+    transition: opacity ease 0.3s;
+    opacity: 0;
+
```
Comment (andrerom) [inline Resources/public/css/theme/views/editpreview.css]: On purpose?

Comment (mhyndle) [inline Resources/public/css/theme/views/editpreview.css]: I guess it's for separate the part that will be changed with transition: https://github.com/ezsystems/PlatformUIBundle/pull/366/files#diff-e5d16b0809fbbf1b577e2b9454cadbf7R27

**Comment by andrerom** [comment]
+1 looks nice :)

File: Resources/public/css/theme/views/editpreview.css (hunk 1 above)
Comment (andrerom) [inline Resources/public/css/theme/views/editpreview.css]: Ok

Comment (dpobel) [inline Resources/public/css/theme/views/editpreview.css]: yep on purpose, just to show/hide the loading animation with a nice transition on the opacity.

Comment (andrerom) [inline Resources/public/css/theme/views/editpreview.css]: It was more the empty new line I was commenting on, but yes :)

Comment (dpobel) [inline Resources/public/css/theme/views/editpreview.css]: I had my glasses but I need more coffee it seems :)

**Pull request merged**